soda_machine.rb

```diff
@@ -7,12 +9,31 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length # or .size
   end
 
   def find_soda(soda_brand)
+    @sodas.find {|soda| soda.brand == soda_brand}
   end
 
   def sell(soda_brand)
```

Comment:
BAH! You're right, totally unnecessary use an instance variable in this case. I have a habit of copying the rspec's naming conventions. Plus the colors, but! Right, unnecessary.